test/performance/odp_packet_gen.c

```diff
@@ -451,6 +512,10 @@ static int open_pktios(test_global_t *global)
 	printf("  tx burst gap        %" PRIu64 " nsec\n",
 	       test_options->gap_nsec);
 	printf("  clock resolution    %" PRIu64 " Hz\n", odp_time_local_res());
+	for (i = 0; i < test_options->num_vlan; i++) {
```

Comment:
VLAN headers are missing from the following pool `seg_len` calculation:
`	seg_len = ODPH_ETHHDR_LEN + ODPH_IPV4HDR_LEN + ODPH_UDPHDR_LEN;`

Reply:
fixed in v2